WordPress/Classes/Networking/PeopleRemote.swift

```diff
@@ -13,21 +13,35 @@ class PeopleRemote: ServiceRemoteWordPressComREST {
         case UnknownError
     }
 
+
+    /// Specifies the number of entities to be retrieved on each query.
+    ///
+    private let pageSize = 50
```

Comment:
Any reason to pick `50` as the page size? I've opted to `20` on Android since it's the default and seemed reasonably high on mobile. On slow connections `50` might take a while.